### PR TITLE
issue: 4249 Change languages we can support and make it a parameter

### PR DESCRIPTION
**Pull request by scottsds**
This isn't my work, I just helped fixed an Arabic related problem. This should be credited to james_. Issue #2059 will be linked to this.

### CODE REVIEW

File: app/views/feedbacks/new.html.erb [hunk 1]
```diff
@@ -19,7 +19,7 @@
...
 
-<p><%= ts("<strong>We can answer Support inquiries in العربية, Bahasa Indonesia, català, čeština, Deutsch, español, français, 한국어, italiano, Kiswahili, magyar, Nederlands, polski, português, Русский, suomi, svenska, Türkçe, 中文.</strong> You can just write your question or feedback in one of the listed languages and we'll sort it out! Please allow for up to one week delay for responses in any language other than English and español.").html_safe %></p>
+<p><strong><%=ts("We can answer Support inquiries in")+' '+ArchiveConfig.SUPPORT_LANGUAGES %></strong>. <%=ts("You can just write your question or feedback in one of the listed languages and we'll sort it out! Please allow for up to one week delay for responses in any language other than English and español.").html_safe %></p>
```
Comment (sarken) [inline app/views/feedbacks/new.html.erb]: We usually use spaces around out + signs for readability: `") + ' ' + ArchiveConfig.SUPPORT_LANGUAGES`

**Comment by sarken** [comment]
@scottsds @zz9pzza I have a question/suggestion regarding this -- would it be possible/feasible/desirable to make it so this was also controlled by the locales form we're adding in issue 4099? That is, instead of having it in the config file, we could add an "Offer support in this locale's language" checkbox to go with the "Use this locale to send email" and "Use this locale for the interface" checkboxes?

**Comment by zz9pzza** [comment]
@sarken yes this would be desirable and quiet easy to do. However given that that hasn't been merged yet I was trying to avoid more pull requests that depending on code that wasn't merged yet. But if it is only a week or two away then reworking this then would be lovely.

**Comment by zz9pzza** [comment]
@sarken thinking about this again, 4099 allows users to select a locale not a language. We offer support in Languages rather than locales. So the change would be in the Languages code not the locale code. Although I think it would look very much like what 4099 has.

**Comment by scottsds** [comment]
@sarken  and @zz9pzza Just let me know what you want me to do with this (including closing it?) and I will. This was orginally james' anyway, I was just helping him fix an Arabic problem

**Comment by zz9pzza** [comment]
I would merge this and work on getting it better later.

File: app/views/feedbacks/new.html.erb [hunk 2]
```diff
@@ -19,7 +19,7 @@
...
 
-<p><%= ts("<strong>We can answer Support inquiries in العربية, Bahasa Indonesia, català, čeština, Deutsch, español, français, 한국어, italiano, Kiswahili, magyar, Nederlands, polski, português, Русский, suomi, svenska, Türkçe, 中文.</strong> You can just write your question or feedback in one of the listed languages and we'll sort it out! Please allow for up to one week delay for responses in any language other than English and español.").html_safe %></p>
+<p><strong><%=ts("We can answer Support inquiries in") + ' '+ ArchiveConfig.SUPPORT_LANGUAGES %></strong>. <%=ts("You can just write your question or feedback in one of the listed languages and we'll sort it out! Please allow for up to one week delay for responses in any language other than English and español.").html_safe %></p>
```
Comment (ariana-paris) [inline app/views/feedbacks/new.html.erb]: Interpolation would make this more flexible for translation in future and more readable. I believe this should work:

```ruby
<%= ts("<strong>We can answer Support inquiries in %{list}.</strong> You can just write your question or feedback in one of the listed languages and we'll sort it out! Please allow for up to one week delay for responses in any language other than English and español.", list: ArchiveConfig.SUPPORT_LANGUAGES).html_safe %>
```

**Comment by scottsds** [comment]
@sarken I have made the suggested changes from @ariana-paris.

**Comment by sarken** [comment]
Alas, we have conflicts! Also, the list will need to be updated to include Tiếng Việt, per the changes for AO3-4327

**Comment by scottsds** [comment]
Merged back into master and updated the config file to include Vietnamese!

**Comment by scottsds** [comment]
2174 is replacing this PR

**Pull request closed without merging**